zipline-loader/src/nativeMain/kotlin/app/cash/zipline/loader/cacheDatabaseNative.kt

```diff
@@ -0,0 +1,16 @@
+package app.cash.zipline.loader
+
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.drivers.native.NativeSqliteDriver
+
+actual class DriverFactory {
+  actual fun createDriver(): SqlDriver {
+    return NativeSqliteDriver(
+      schema = Database.Schema,
+      name = "zipline-loader.db",
+    )
+  }
+}
+
+// TODO find the native exception class
```

Comment:
File an issue so we don't forget? Or are you planning on doing this before merge.

Reply:
Good call: https://github.com/cashapp/zipline/issues/433